web/src/app/util/copyToClipboard-v2.js

```diff
@@ -0,0 +1,16 @@
+export default text => {
+  if (!navigator.clipboard) {
+    window.alert(
```

Comment:
Don't use `alert`

Reply:
If anything, it should probably be a MUI dialog rather than an alert. Browsers can block alerts, and they are generally frowned upon as they can't really be styled effectively either